oat-sa/tao-core
# Task/aut 875/data testid attrs

**Ticket:** https://oat-sa.atlassian.net/browse/AUT-949

**How to test:**
- Create item (test, asset) class
- Add properties
- Check that properties controls (inputs, textareas, etc.) have data-testid attrs
- Check that e2e tests run successfully

_Export button was not affected as it's associated with changes in  the ui libs (like [here](https://github.com/oat-sa/tao-core-ui-fe/blob/master/src/loadingButton/tpl/button.tpl))_

**Deployed on:** http://test-anastasia-2.playground.kitchen.it.taocloud.org:45401
